shepherd/Shepherd.py

```diff
@@ -18,7 +18,7 @@ def __init__(self, matchNumber):
         self.alliances= {'blue': Alliance(teams[0], teams[1]), 'gold': Alliance(teams[2], teams[3])}
         self.currentStage = 0
 
-        #The follwing are elements that are configured 
+        #The follwing are elements that are configured
```

Comment:
s/follwing/following

Reply:
We'll fix this later when this file is actually worked on (I hope we remember)